rust/datafusion/src/test/mod.rs

```diff
@@ -106,137 +106,7 @@ pub fn aggr_test_schema() -> SchemaRef {
     ]))
 }
 
-/// Format a batch as csv
-pub fn format_batch(batch: &RecordBatch) -> Vec<String> {
```

Comment:
format batch is similar, but not the same, as pretty_print

The major difference is that it renders NULL values as `"NULL"` whereas the pretty printer renders them as an empty string `""`